packages/bpk-docs/src/pages/TooltipsPage/TooltipsPage.js

```diff
@@ -129,6 +129,17 @@ const TooltipsPage = () => (<DocsPageBuilder
   ]}
   components={components}
   readme={tooltipReadme}
+  usageTable={{
+    dos: [
+      'To provide additional information about an element on the page, shown on hover.',
+      'Text-only works best.',
+    ],
+    donts: [
+      'For touch devices (use a popover instead).',
+      'For long or complex content or when content includes interaction (use a popover or modal).',
+      'On interactive elements that can receive focus such as links, buttons and inputs.',
+    ],
+  }}
```

Comment:
I had a look at this on the site and I think it might read better if it used more complete sentences, e.g.:

> (Don't) Use on touch devices (use a popover instead)

![screen shot 2017-12-15 at 16 32 21](https://user-images.githubusercontent.com/73652/34051261-ca48c592-e1b5-11e7-8d4e-8bee27a4836c.png)

Reply:
Yeh, good shout. Let me revise.